katello.te

```diff
@@ -63,35 +61,8 @@ allow foreman_rails_t amqp_port_t:tcp_socket name_connect;
 # Katello uses certs in /etc/pki/katello for websockets
 miscfiles_read_certs(websockify_t)
 
-# Katello can be configured to read from pulp's published dir
-optional_policy(`
-    apache_manage_sys_content(foreman_rails_t)
-    apache_manage_sys_content_rw(foreman_rails_t)
-')
-
 # Installation generates load denial
 require {
   type load_policy_t;
 }
 userdom_write_inherited_user_tmp_files(load_policy_t)
-
-# Allow the ping controller to check pulpcore-api service
-# https://projects.theforeman.org/issues/31169
-allow foreman_rails_t systemd_unit_file_t:dir search;
```

Comment:
Just for my own sanity, this is not required anymore since https://github.com/Katello/katello/commit/b5a79de628a4114106855925b5e9cb26cffd60c3